ProcessLib/LiquidFlow/LiquidFlowProcess.h

```diff
@@ -72,6 +77,14 @@ class LiquidFlowProcess final : public Process
                                           GlobalVector const& x) override;
 
     bool isLinear() const override { return true; }
+
+    std::vector<MaterialLib::Fluid::FluidProperties*> getFluidProperties() const
+    {
+        std::vector<MaterialLib::Fluid::FluidProperties*> material_properties =
+            {_material_properties->getFluidProperties()};
```

Comment:
Originally it is returned in a vector in case to have a same return turn type with multiphase flow process, which can have more than one fluids. Now function is changed to
```
    MaterialLib::Fluid::FluidProperties* getFluidProperties() const
    {
        return _material_properties->getFluidProperties();
    }
```
for the LiquidFlow only.